new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMark.java

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) 2020, Oracle Corporation and/or its affiliates.
+// Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+package oracle.weblogic.kubernetes;
+
+import io.kubernetes.client.openapi.ApiException;
+import oracle.weblogic.kubernetes.annotations.IntegrationTest;
+import oracle.weblogic.kubernetes.extensions.LoggedTest;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static oracle.weblogic.kubernetes.actions.TestActions.createSecret;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatCode;
+import static org.assertj.core.api.Assertions.catchThrowable;
+
+@DisplayName("Simple validation of basic domain functions")
+@IntegrationTest
+class ItMark implements LoggedTest {
+
+  @Test
+  @DisplayName("Test something")
```

Comment:
Another random nit, unrelated to this check-in, but still useful, would be a built-in 'fail fast' option.  This is useful for debugging.

The old operator tests don't seem to have one - at least I couldn't get one from QA, or just plain forgot, and maybe was a bit too lazy to spend much time researching one.  This forced me to hand code a temporary System.exit on failure, or to sit there, watch the test run, and ^C in order to stop the test and leave all resources up/running and debuggable at the point of failure.

Maybe JUnit 5 has one?  I couldn't find one.  I did find [this](http://www.jeeatwork.com/?p=246) after some googling, which lets you replace the JUnit failure handler and have it call System.exit.  If JUnit5 doesn't have something built in, we could modify this solution slightly and only have it activate itself if some easy-to-pass-in global option comes from the command line.

Reply:
That's an interesting point- and one where i think a big global switch would make sense.  I understand if you are running the suite on your machine, and it fails, you would like it to stop immediately so you can go look at the environment.  But at the same time, when I am running it in Jenkins, I want it to go all the way through and run all the tests no matter what. 
Interestingly Ginkgo, which we use on other projects, has a nice global switch called `-keepGoing` that provides exactly that behavior!